salt/orch/update-etc-hosts.sls

```diff
@@ -1,4 +1,4 @@
-{%- set updates_all_target = 'P@roles:(admin|kube-(master|minion)) and G@bootstrap_complete:true and not G@bootstrap_in_progress:true and not G@update_in_progress:true' %}
+{%- set updates_all_target = 'P@roles:(admin|kube-(master|minion)) and G@bootstrap_complete:true and not ( G@bootstrap_in_progress:true or G@update_in_progress:true or G@removal_in_progress:true )' %}
```

Comment:
Unrelated to your PR: This is getting out of hand - we need to find a better solution.

Reply:
Indeed. :-)

Reply:
It can be _reduced_ to:

```
'P@roles:(admin|kube-(master|minion)) and G@bootstrap_complete:true and not P@*_in_progress:true'
```